app/models/project.rb

```diff
@@ -11,5 +11,6 @@ class Project < ActiveRecord::Base
     :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
   }
 
+  validates_attachment_content_type :snap_file, :content_type => 'text/xml'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.